safekeeper/src/callmemaybe.rs

```diff
@@ -48,7 +48,7 @@ async fn request_callback(
     // it is required to correctly manage callmemaybe subscriptions when more than one pageserver is involved
     // TODO it is better to use some sort of a unique id instead of connection string, see https://github.com/zenithdb/zenith/issues/1105
     let callme = format!(
-        "callmemaybe {} {} host={} port={} options='-c ztimelineid={} ztenantid={} pageserver_connstr={}'",
+        "callmemaybe {} {} host={} port={} options='-c ztimelineid={} TenantId={} pageserver_connstr={}'",
```

Comment:
> hmm, not too sure why the perf tests fail here thinking . @hlinnaka, @lubennikovaav do you have any suggestions?

Because of this change, maybe? The option in safekeeper is still 'ztenantid'. There are a few other cases like this in this PR, too, where the 'ztenantid' is an option name in a string.

Reply:
thank you for catching that. my bad - i'll review again, and update.